### DISCUSSION TITLE
Remix 1.18 deployed on Vercel keeps reloading the page on Shopify

### DISCUSSION THREAD

**Discussion by nullndr** [remix-run/remix#6732] [Q&A]
I update a remix project deployed on Vercel for a Shopify app with the latest version (1.18).

When I open the app it keeps reloading with the following error:

```
Initial URL 
(/?host=...&hmac=...&embedded=1&locale=en-US&timestamp=1688116939&session=...&shop=...) 
does not match URL at time of hydration 
(/?embedded=1&hmac=...&host=...&locale=en-US&session=...&shop=...&timestamp=1688116939), 
reloading page...
```

This check has been introduced with https://github.com/remix-run/remix/pull/6409.

Does anybody know if there is a workaround for this?

**Comment by jezzzm** [comment]
i have been tearing my hair out over this one. also on vercel.

when hydrated, the order of the search params is sometimes changed. not sure what causes that, but we end up in a redirect loop.

would appreciate some eyes on this, thanks!

**Reply by nullndr** [reply]
Hey, this has been resolved on `1.18.1`
